app/src/main/java/ch/epfl/sdp/blindly/location/LocationService.kt

```diff
@@ -0,0 +1,9 @@
+package ch.epfl.sdp.blindly.location
+
+import android.location.Location
+import android.location.LocationListener
+
+interface LocationService: LocationListener {
+
+    fun getCurrentLocation(): Location?
+}
```

Comment:
Is this interface relevant ?

Reply:
If we want to add another location service it would be easier since it is more modular